### PR TITLE
Prison envirosuits!

### PR DESCRIPTION
**Pull request by AMyriad**
# Document the changes in your pull request
This adds a subtype of the normal envirosuit intended for plasmamen prisoners.
It spawns in every brig locker along regular prison clothes, and has its sensors locked, etc, to match.

# Why is this good for the game?
Whenever I'm jailed for war crimes, I wish I had that orange drip to fit in with my prison gang. Being stuck with the colorful suit on top of the helmet feels and looks weird, and I dun like it.
Also whenever I jail a plasmaman, I realize they have the advantage of being able to keep their sensors off, unlike normal prisoners who are forced to keep them on tracking or be naked. Now plasmamen can suffer more with their permanently ON sensors :D

# Testing
image

# Spriting
Designed to be similar to the standard suit, but with:
- Black markings to stand out even more/look more prisoner-like
- Color tweaks to match prison jumpsuits/shoes
image
image

# Changelog
:cl:  
rscadd: Added envirosuits designed for prisoners, found in brig lockers along with normal prison wear
imageadd: Prisoner envirosuit sprites
mapping: Added a prison envirosuit to Box permabrig's holding cell alongside the prison uniforms
/:cl:

### CODE REVIEW

**Comment by azzzertyy** [comment]
can you add a shower nearby it so that it isn't a pain in the ass to apply it?

**Review by ToasterBiome — APPROVED** [review]

**Comment by AMyriad** [comment]
> can you add a shower nearby it so that it isn't a pain in the ass to apply it?

there are showers inside the prison, where I imagine officers would prefer to strip inmates in rather than outside

**Comment by AMyriad** [comment]
P.S. don't merge this until the AI rework thing is merged

**Pull request merged**
